### ISSUE TITLE
words extraction is not happening

### ISSUE THREAD

**Issue by ashishbasantani** [Breta01/handwriting-ocr#70]
I cannot find words-final folder, & I am unable to extract data.

**Comment by ashishbasantani** [comment]
"import enchant error" as enchant is not available for 64 bit OS so what can be other option to install enchant

**Comment by Breta01** [comment]
What did you try to install? You should run: `pip install pyenchant`.

I just found out, that it's no longer under maintenance. I will try to replace it.

**Comment by ashishbasantani** [comment]
Getting error while executing OCR.ipynb file "TypeError: Cannot interpret feed_dict key as Tensor: The name 'x:0' refers to a Tensor which does not exist. The operation, 'x', does not exist in the graph."

**Comment by Breta01** [comment]
Yes, it is because CTC model requires little bit different inputs. OCR notebook right now works only with char classifier. Take a look into `ocr_evaluator.ipynb` at `class WordCycler()`. I hope you will be able to figure out neccesseray changes.

Once I have some time, I will extend the OCR notebook to support all models.
